### PR TITLE
refactor: assign ordinals automatically

### PR DESCRIPTION
**Pull request by KSXGitHub**

### CODE REVIEW

File: lockfile/lockfile-file/src/sortLockfileKeys.ts [hunk 1]
```diff
@@ -31,23 +31,24 @@ const ORDERED_KEYS = {
...
+type RootKey = keyof LockfileFile
+const ROOT_KEYS: readonly RootKey[] = [
+  'lockfileVersion',
+  'settings',
+  'neverBuiltDependencies',
+  'onlyBuiltDependencies',
+  'overrides',
+  'packageExtensionsChecksum',
+  'patchedDependencies',
+  'specifiers' as string as RootKey, // TypeScript says that 'specifiers' doesn't exist in LockfileFile, should it be removed?
```
Comment (zkochan) [inline lockfile/lockfile-file/src/sortLockfileKeys.ts]: yes, it can be removed.

Comment (KSXGitHub) [reply lockfile/lockfile-file/src/sortLockfileKeys.ts]:
```suggestion
```

File: .gitattributes [hunk 2]
```diff
@@ -1,3 +1,2 @@
 * text eol=lf
-
-*.tgz binary
+*.tgz filter=lfs diff=lfs merge=lfs -text
```
Comment (KSXGitHub) [inline .gitattributes]: @zkochan `lfs` appears to not have been removed here. Should we be worried?

Comment (zkochan) [reply .gitattributes]: can you remove the first 2 commits in this PR?

**Review by zkochan — APPROVED** [review]

**Pull request merged**